dotCMS/src/main/java/com/dotmarketing/business/RelationshipAPIImpl.java

```diff
@@ -496,23 +508,35 @@ private void migrateOldRelationshipReferences(Relationship oldRelationship,
         dc.loadResult();
 
         //Delete the old relationship
-        APILocator.getRelationshipAPI().delete(oldRelationship);
+        //APILocator.getRelationshipAPI().delete(oldRelationship);
 
         //Reindex both Content Types, so the content show the relationships
         contentletAPI.refresh(oldRelationship.getParentStructure());
         contentletAPI.refresh(oldRelationship.getChildStructure());
     }
 
+    /**
+     * This creates a relationship field but DOES NOT create try to create the underlying relationship
+     * @param fieldName
+     * @param parentContentTypeID
+     * @param cardinality
+     * @param isRequired
+     * @param childContentTypeVariable
+     * @return
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
     private com.dotcms.contenttype.model.field.Field createRelationshipField(final String fieldName, final String parentContentTypeID, final int cardinality, final boolean isRequired, final String childContentTypeVariable)
             throws DotDataException, DotSecurityException {
-        final com.dotcms.contenttype.model.field.Field field = FieldBuilder.builder(RelationshipField.class)
+        final com.dotcms.contenttype.model.field.RelationshipField field = ImmutableRelationshipField.builder()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'com.dotcms.contenttype.model.field.RelationshipField' due to existing import 'com.dotcms.contenttype.model.field.RelationshipField'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5653044)